tests/integration/api_packages_debian_test.go

```diff
@@ -144,6 +147,14 @@ func TestPackageDebian(t *testing.T) {
 								}
 								return seen
 							})
+
+							req = NewRequestWithBody(t, "PUT", uploadURL, createArchive(packageName, packageVersion, architecture, false))
+							AddBasicAuthHeader(req, user.Name)
+							MakeRequest(t, req, http.StatusBadRequest)
+
+							req = NewRequestWithBody(t, "PUT", uploadURL, createArchive(packageName, packageVersion, architecture, true))
+							AddBasicAuthHeader(req, user.Name)
+							MakeRequest(t, req, http.StatusBadRequest)
```

Comment:
Isn't there a test missing for an archive with trailing slashes that succeeds?

Reply:
In theory yes but the new test indicates a successfull upload. The duplicate package error is raised after processing the file while inserting it into the database.